src/client/components/UserHeader.js

```diff
@@ -103,7 +104,8 @@ const UserHeader = ({
               />
             </div>
           </div>
-          {isFollowing &&
+          {authenticated &&
+            isFollowing &&
```

Comment:
I think `isFollowing` should take whether user is logged in or not in account. Can we update how `isFollowing` is calculated, so `isFollowing` really means that user is following me.